packages/amplify_core/lib/src/types/storage/download_data_options.dart

```diff
@@ -4,11 +4,16 @@
 import 'base/storage_operation_options.dart';
 
 /// {@template amplify_core.storage.download_data_options}
-/// Configurable options to initiate a [StorageDownloadDataRequest].
+/// Configurable options for `Amplify.Storage.downloadData`.
 /// {@endtemplate}
-class StorageDownloadDataOptions extends StorageOperationOptions {
+class StorageDownloadDataOptions<PluginOptions extends Object?>
```

Comment:
Let's avoid generics. They're underpowered footguns in Dart unfortunately (as we've learned 😅)

Reply:
this is based on API change review proposal unless I'm misreading the proposal